src/api/InfluxDB.cpp

```diff
@@ -245,7 +262,8 @@ void vz::api::InfluxDB::send()
 			curl_easy_setopt(_api.curl, CURLOPT_PASSWORD, _password.c_str());
 		}
 		curl_easy_setopt(_api.curl, CURLOPT_URL, _url.c_str());
-		curl_easy_setopt(_api.curl, CURLOPT_VERBOSE, options.verbosity());
+		curl_easy_setopt(_api.curl, CURLOPT_VERBOSE, options.verbosity() > 0);
+		curl_easy_setopt(_api.curl, CURLOPT_SSL_VERIFYPEER, 0L);
```

Comment:
adding this with an option would be nicer (if option_... then ...)

Reply:
sorry, I might have been a bit more specific: I actually thought that adding an option (similar to send_uuid) like "ssl_verifypeer" to decide whether the ssl_verifypeer should be disabled or not.

Reply:
Ok. Now I hopefully understood you right. Fixed so far.